app/models/tasks/copy_media_task.rb

```diff
@@ -22,7 +22,7 @@ def task_options
   end
 
   def source_url(media_resource)
-    media_resource.original_url.sub(/\?.*$/, "")
+    media_resource.href.sub(/\?.*$/, "")
```

Comment:
So this "strip query params" was added in #135.  I'm not entirely clear if/why we'd still need it?  But left it in for now.

Reply:
Biggo Shruggo.
What could potentially happen if it's removed?

Reply:
Probably better to leave it in then, if it's not hurting anything and adds some use.